lib/galaxy/tools/test.py

```diff
@@ -157,7 +157,8 @@ def __process_raw_inputs( self, tool_inputs, raw_inputs, parent_context=None ):
                     for input_name, input_value in case.inputs.items():
                         case_inputs = self.__process_raw_inputs( { input_name: input_value }, raw_inputs, parent_context=cond_context )
                         expanded_inputs.update( case_inputs )
-                    expanded_case_value = self.__split_if_str( case.value )
+                        if not value.type == "text":
```

Comment:
Well, the original code always calls `self.__split_if_str( case.value )`, which I would like to avoid if `value.type` is `text`.
You're suggesting to put `if not value.type == "text":` outside of the for loop?
I don't think that would work?!
Maybe I don't understand the question, though.